app/views/layouts/authentication.html.slim

```diff
@@ -1,15 +1,31 @@
 doctype html
 html
   head
-    title Velum
-    = stylesheet_link_tag 'application', media: 'all'
+    title SUSE CaaSP: Velum
+    = stylesheet_link_tag 'authentication', media: 'all'
     = javascript_include_tag 'application'
     = csrf_meta_tags
     meta content="width=device-width, initial-scale=1" name="viewport"
     meta content="Velum" name="apple-mobile-web-app-title"
     meta content="Velum" name="application-name"
-    meta content="#da532c" name="msapplication-TileColor"
-    meta content="#205683" name="theme-color"
-  body.login
-    .container-fluid
-      = yield
+  body.dark-login
+    header
+      .container
+        .row
+          .col-md-12
+            img.logo[src='/logo.svg']
```

Comment:
That was just a way I've found to make it work after dealing for a while.
Please, feel free to fix it to a more proper way